myhoard/basebackup_operation.py

```diff
@@ -288,6 +289,8 @@ def _process_input_output(self):
 
         if exit_code != 0:
             self.log.error("xtrabackup exited with non-zero exit code %s: %r", exit_code, pending_output)
+            if self.has_block_mismatch:
+                raise BlockMismatchError(f"xtrabackup failed with code {exit_code} due log block mismatch.")
```

Comment:
**comment** Had to track separately the block mismatch error from other output line, xtrabackup will finish with a different output that those not indicates this as root cause.